src/I18n/DateFormatTrait.php

```diff
@@ -257,9 +316,10 @@ public static function setJsonEncodeFormat($format)
      *
      * @param string $time The time string to parse.
      * @param string|array|null $format Any format accepted by IntlDateFormatter.
+     * @param string|\DateTimeZone|null $timezone Timezone string or DateTimeZone object
      * @return static|null
      */
-    public static function parseDateTime($time, $format = null)
+    public static function parseDateTime($time, $format = null, $timezone = null)
```

Comment:
The method used timezones internally already without yielding control to the user. You could specify a timezone of a time object that is not UTC and hopefully you can work with that? At least that was my idea. No good?